hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-timeline-pluginstorage/src/test/java/org/apache/hadoop/yarn/server/timeline/TestEntityGroupFSTimelineStore.java

```diff
@@ -173,7 +178,7 @@ public void setup() throws Exception {
     }
 
     store = new EntityGroupFSTimelineStore();
-    if (currTestName.getMethodName().contains("Plugin")) {
+    if (testInfo.getDisplayName().contains("Plugin")) {
```

Comment:
We need only the method name, so `testInfo.getTestMethod().getName()` seems more informative.

Reply:
There is no such function `getName()` from `testInfo.getTestMethod()`.

Reply:
And output of `currTestName.getMethodName()` gives output as `testPluginRead()` for `testPluginRead` method

Reply:
Oh, sorry I missed the return value is wrapped by `Optional`. Probably `testInfo.getTestMethod().get().getName()`?

Reply:
Makes sense, Addressed in my latest commit.

Reply:
Thanks a lot!